test/runtime/Policies.go

```diff
@@ -874,6 +874,54 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 
 	})
 
+	It("Enforces ToFQDNs policy", func() {
+		By("Importing policy with ToFQDN rules")
+		// notaname.cilium.io never returns IPs, and is there to test that the
+		// other name does get populated.
+		fqdnPolicy := `
+[
+  {
+    "endpointSelector": {
+      "matchLabels": {
+        "container:id.app1": ""
+      }
+    },
+    "egress": [
+      {
+				"toPorts": [{
+          	"ports":[{"port": "53", "protocol": "ANY"}]
+        }]
+      },
+      {
+        "toFQDNs": [
+          {
+            "matchName": "cilium.io"
+          },
+          {
+            "matchName": "notaname.cilium.io"
+          }
+        ]
+      }
+    ]
+  }
+]`
+		_, err := vm.PolicyRenderAndImport(fqdnPolicy)
+		Expect(err).To(BeNil(), "Unable to import policy: %s", err)
+
+		By("Testing that ToFQDN enables policy enforcement")
+
+		By("Denying egress to IPs of non-ToFQDN DNS names that are looked up")
+		for _, blockedTarget := range []string{"www.cilium.io", "google.com", "yahoo.com", "1.1.1.1", "127.0.0.1"} {
```

Comment:
Are all these targets needed? Each one will take Curl Max timeout (3 seconds) so not sure that we need to test all. 

On the other hand, I would add here another test, something like `cilium.io.customdomain.io` that looks like as another corner case.

Reply:
you're right. They're actually pretty redundant. I'll use `www.cilium.io` (exists but should block) and `cilium.io.cilium.io` (doesn't exist and should block, and it tests if we somehow break if the domain repeats). I need an actual IP in there, though, just as a sanity check. 

Including your suggestion, there are 3 negative tests, each taking 3s. Is that ok?